powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/response/GetActivationStatusResponse.java

```diff
@@ -45,6 +45,8 @@ public class GetActivationStatusResponse {
     private String platform;
     private String deviceInfo;
     private String applicationId;
+    private Long failedAttempts;
```

Comment:
Shouldn't it be mentioned in `WebServices-Methods.md`?

Reply:
Good catch